runtime/lua/vim/treesitter/query.lua

```diff
@@ -1028,7 +1028,10 @@ end
 ---
 --- @param lang? string language to open the query editor for. If omitted, inferred from the current buffer's filetype.
 function M.edit(lang)
-  vim.treesitter.dev.edit_query(lang)
+  local _, err_msg = vim.treesitter.dev.edit_query(lang)
```

Comment:
assert?

Reply:
I thought about that but I think it cannot be done here since `edit_query` returns `nil` even upon success (meaning the assertion would always fail). That's why I think it should return true on success but maybe could be done in a small follow-up PR? To lubricate the merging of this one

Reply:
Please add a todo then. (And let's follow up quickly ;))